catalogue/webapp/test/utils/dates.test.ts

```diff
@@ -128,23 +128,23 @@ describe('findClosedDays', () => {
   });
 });
 
-describe('findNextRegularOpenDay: finds the earliest date on which the venue is normally open', () => {
+describe('findNextPickUpDay: finds the earliest date on which requested items can be picked up', () => {
   it('returns the same date provided if it occurs on one of the regular open days', () => {
-    const result = findNextRegularOpenDay(
+    const result = findNextPickUpDay(
       london('2022-01-15'), // Saturday
       [0, 1, 2] // Sunday, Monday, Tuesday
     );
     expect(result.toDate()).toEqual(london('2022-01-15').toDate()); // Saturday
   });
-  it('returns the date of the next regular open day, if the date provided is a regular closed day', () => {
-    const result = findNextRegularOpenDay(
+  it('returns the date of the next regular open day plus one, if the date provided is a regular closed day', () => {
```

Comment:
Let's express this in terms of what it means for LE&E, not the exact logic:

```suggestion
  it('leaves a full working day between the request and retrieval', () => {
```